components/dot_menu/dot_menu.jsx

```diff
@@ -11,7 +11,7 @@ import Permissions from 'mattermost-redux/constants/permissions';
 import {showGetPostLinkModal} from 'actions/global_actions.jsx';
 import {Locations, ModalIdentifiers, UNSET_POST_EDIT_TIME_LIMIT} from 'utils/constants.jsx';
 import DeletePostModal from 'components/delete_post_modal';
-import DelayedAction from 'utils/delayed_action.jsx';
```

Comment:
It's better to not supply extensions at all. It's already supported by webpack and it looks cleaner :)